### DISCUSSION TITLE
After upgrade, storybook no longer shows MDX documents

### DISCUSSION THREAD

**Discussion by jwir3** [storybookjs/storybook#20237] [Help]
I have the following directory setup for my stories:

  - src/
    - stories
      - Components/
      - Layouts/
      - Introduction.stories.mdx

Inside of `Introduction.stories.mdx`, I have some base information for new users to the project. It used to display this as a landing page when my project was launched through `start-storybook`, but since I upgraded to `6.5.14` (I believe I was at 6.3.something prior), I now only see the Components, without the two .mdx stories I have, as in this screenshot:

![image](https://user-images.githubusercontent.com/997106/207462188-611bd7b9-b43c-4fae-95c0-22f38e580ee2.png)

I feel like there is a setting I'm missing to get the .stories.mdx files to be rendered that's new in this new version. Can someone give me some advice on how I might troubleshoot this issue?

**Comment by shilman** [comment]
Are there any errors in the browser console?

You can also view source in chrome dev tools.

Happy to look at a reproduction if you have one public

**Comment by jwir3** [comment]
Hi @shilman:

Thanks for the response. There doesn't appear to be any errors in the browser console, aside from one stating that there was an interruption in the connection:

```
The connection to http://localhost:6006/__webpack_hmr was interrupted while the page was loading. [client.js:89](webpack://foamfactory/aegir/node_modules/webpack-hot-middleware/client.js)
InstallTrigger is deprecated and will be removed in the future. [902.f3fe1f4644e1c0685d00.manager.bundle.js:2:1559663](https://aegir.foamfactory.io/902.f3fe1f4644e1c0685d00.manager.bundle.js)
InstallTrigger is deprecated and will be removed in the future. [137.4d5ca6ff.iframe.bundle.js:2:1415701](https://aegir.foamfactory.io/137.4d5ca6ff.iframe.bundle.js)
[content.js:46:1077125](moz-extension://3c98729d-6afd-4801-810c-4caadf363211/content.js)
```

I have a production build available at http://aegir.foamfactory.io. Let me see if I can post a development build quickly. When I run it locally, it also doesn't appear to have any errors in the console, aside from some issues where I didn't specify a `key` prop for a list element.
